samples/gke_enable_stackdriver_logging.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: GCPGKEEnableStackdriverLoggingConstraintV1
+metadata:
+  name: enable_gke_stackdriver_logging
+  annotations:
+    description: Ensure stackdriver logging is enabled on a GKE cluster
+    benchmark: CIS11_7.02
```

Comment:
Can you update this to the latest format: https://github.com/forseti-security/policy-library/blob/ec9b5c2040d9afe2a551d8b78505eb97a55d376a/samples/gke_node_pool_auto_repair.yaml#L22

Please also make sure this is CIS 7.02